src/routes/grantform/GrantForm.css

```diff
@@ -15,14 +20,140 @@
   margin-bottom: 40px;
 }
 
-.subHeading {
-  font-size: 1.5em;
+.labelContainer {
+  margin-top: 40px;
+  margin-bottom: 10px;
+}
+
+.label {
+  font-size: 16px;
+  font-family: 'RisingSunDemiBold', sans-serif;
+  color: #95989a;
+}
+
+.inputError {
+  border: 1px solid #ffc14b !important;
+}
+
+.inputErrorMessage {
+  color: #fff;
+  background-color: #ffc14b;
+  padding: 10px;
+}
+
+.input {
+  height: 40px;
+  padding-left: 10px;
+  border: 1px solid #c6cacc;
+  border-top: 0.1px solid #c6cacc;
+  border-radius: 1px;
+  outline: none;
+  background: #fafafa;
+  width: 100%;
+  font-size: 15px;
+  font-family: 'RisingSunDemiBold', sans-serif;
+  color: #808080;
+  box-sizing: border-box;
+}
+
+.input::-webkit-input-placeholder {
+  color: #95989a;
+  font-family: 'RisingSunRegular', sans-serif;
+}
+
+.input::-moz-placeholder {
+  color: #95989a;
+  font-family: 'RisingSunRegular', sans-serif;
+}
+
+.input:-ms-input-placeholder {
+  color: #95989a;
+  font-family: 'RisingSunRegular', sans-serif;
+}
+
+.input:-moz-placeholder {
+  color: #95989a;
+  font-family: 'RisingSunRegular', sans-serif;
+}
+
+.contactContainer {
+  display: flex;
+  margin-top: 15px;
+}
+
+.email {
+  flex: 27;
+  margin-right: 10px;
+}
+
+.phone {
+  flex: 16;
 }
 
-.nextSteps {
-  margin: 10px 0 0;
+.description {
+  margin-top: 10px;
+  padding-top: 10px;
+  height: 180px;
 }
 
-.step {
+.date {
+  width: 120px;
   margin-bottom: 15px;
 }
+
+.costContainer {
+  display: flex;
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.ask {
+  flex: 1;
+  margin-right: 10px;
+}
+
+.totalCost {
+  flex: 1;
+}
+
+.sendContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 40px;
+}
+
+.captchaContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.captcha {
+  margin-top: 20px;
+}
+
+.send {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 220px;
+  height: 50px;
+  margin-top: -25px;
+  background: #2677c2;
+  border-radius: 4px;
+  font-family: 'RisingSunDemiBold', sans-serif;
+  font-size: 16px;
+  color: #fff;
+  cursor: pointer;
+}
+
+.formErrorMessage {
+  padding-top: 10px;
+  text-align: center;
+}
+
+/* .success {
+  width: 84px;
+  height: 40px;
+} */
```

Comment:
Lol, what a derp on my part. Must have made this error in the final stage when the linter was complaining about unused styles and I split the css into two files.